app/policies/news_policy.rb

```diff
@@ -0,0 +1,21 @@
+class NewsPolicy < AdminPolicy
+  def index?
+    current_user.admin? || current_user.lead? || current_user.members? if current_user
```

Comment:
['admin', 'lead', 'member'].include?(current_user.role) - you can do something like this instead a lot of 'or' operands